Gulpfile.js

```diff
@@ -216,3 +216,9 @@ gulp.task('test-client', ['build'], function () {
 });
 
 gulp.task('test', ['lint', 'test-server']);
+
+gulp.task('error-message-lister', function () {
```

Comment:
Yep, it's just shows you how all errors will look like in the given reporter in TTY.